apps/website/docs/react/api/make-styles.md

```diff
@@ -260,6 +260,37 @@ const useClasses = makeStyles({
 });
 ```
 
+### `null` values
```

Comment:
There should also be a playground for this example

Reply:
As it relies on `mergeClasses()`, I am unsure about playground example